src/NServiceBus.AcceptanceTests/Core/Pipeline/When_discontinuing_message_dispatch.cs

```diff
@@ -1,11 +1,28 @@
-﻿namespace NServiceBus.AcceptanceTests.Sagas
+﻿namespace NServiceBus.AcceptanceTests.Core.Pipeline
 {
+    using System;
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using EndpointTemplates;
+    using NUnit.Framework;
 
-    public class When_receiving_that_should_start_a_saga : NServiceBusAcceptanceTest
+    public class When_discontinuing_message_dispatch : NServiceBusAcceptanceTest
     {
+        [Test]
+        public async Task Should_not_continue_to_dispatch_the_message()
+        {
+            var context = await Scenario.Define<SagaEndpointContext>(c => { c.InterceptSaga = true; })
```

Comment:
I think the `InterceptSaga` is no longer required and can be hard-coded entirely.